forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -3210,6 +3210,42 @@ else if (game.getRules().hasAppliedVariant(GameType.Oathbreaker)) {
         return eff;
     }
 
+    public void createPlanechaseEffects(Game game) {
+        final PlayerZone com = getZone(ZoneType.Command);
+        final String name = "Planar Dice";
+        final Card eff = new Card(game.nextCardId(), game);
+        eff.setTimestamp(game.getNextTimestamp());
+        eff.setName(name);
+        eff.setOwner(this);
+        eff.setImmutable(true);
+        String image = ImageKeys.getTokenKey("planechase");
+        eff.setImageKey(image);
+
+        String trigger = "Mode$ PlanarDice | Result$ Planeswalk | TriggerZones$ Command | ValidPlayer$ You | Secondary$ True | " +
+                "TriggerDescription$ Whenever you roll the Planeswalker symbol on the planar die, planeswalk.";
+        String rolledWalk = "DB$ Planeswalk | Cause$ PlanarDie";
+        Trigger planesWalkTrigger = TriggerHandler.parseTrigger(trigger, eff, true);
+        planesWalkTrigger.setOverridingAbility(AbilityFactory.getAbility(rolledWalk, eff));
+        eff.addTrigger(planesWalkTrigger);
+
+        String chaosTrig = "Mode$ PlanarDice | ValidPlayer$ You | Result$ Chaos | TriggerZones$ Command | Static$ True";
+        String rolledChaos = "DB$ ChaosEnsues";
+        Trigger chaosTrigger = TriggerHandler.parseTrigger(chaosTrig, eff, true);
+        chaosTrigger.setOverridingAbility(AbilityFactory.getAbility(rolledChaos, eff));
+        eff.addTrigger(chaosTrigger);
```

Comment:
maybe remove this trigger and add just:
`getTriggerHandler().runTrigger(TriggerType.ChaosEnsues, runParams, false);`

in the PlanarDice class?